### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by marijanbrvar**
Add navigation with the menu to the web app.
1. Add class for Nav
2. Add Contact section
3. Add Footer
4. Add date and clock for current session

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -70,18 +103,51 @@ class Screen {
...
+    <div class="Header-item mr-0">
+      <a class="Header-link mr-3" data-target="#contact">Contact</a>
     </div>
     `;
 
-    this.layoutMain = document.createElement('div');
+    this.clock = document.createElement('div');
+    this.clock.classList.add('clock');
+    this.clock.innerHTML = `
+    ${Date()}
```
Comment (ZawadiSumba66) [inline app.js]: - [x] For setting the time you were supposed to use the luxon library. However, I have noticed that the link is not working on the project requirements but we apologize for the inconvenience. Please refer to this [article](https://moment.github.io/luxon/#/) on how to set it up as the broken link gets updated. Unfortunately, this is a hard requirement for the project and needs to be implemented.

File: app.js [hunk 2]
```diff
@@ -61,6 +61,39 @@ class UI {
...
+        if (e.target.tagName === 'A') {
+          this.toggleTabs(e);
+          this.toggleContent(e);
+        }
+      });
+    });
+  }
+
+  toggleTabs(e) {
+    // remove current active classes
```
Comment (ZawadiSumba66) [inline app.js]: - [x] Please remove comments from your code. For a cleaner code, we encourage avoiding them.

**Review by ZawadiSumba66 — CHANGES_REQUESTED** [review]
#### Hi @marijanbrvar  && @moise10r 👋 👋  , 

Good job so far!

### HIGHLIGHTS 🚀 🚀 

- Great UI design 👍 
- Well-organized code.
- Descriptive README file.

However, there are some issues that you still need to work on to move to the next step but you are almost there!

## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by smcommits — APPROVED** [review]
### Status: Approved 👍🏽 

Hi @moise10r, 

## You have done a great job! Your project is complete. This PR is now approved. You can go ahead and merge the branch. Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**